docs/modules/ROOT/pages/advanced-how-tos/managing-multiple-versions.adoc

```diff
@@ -63,53 +62,101 @@ spec:
-      finalizers:
-      - test.appstudio.openshift.io/component
-      - component.appstudio.redhat.com/finalizer
-      - image-controller.appstudio.openshift.io/image-repository
-      - image-registry-secret-sa-link.component.appstudio.openshift.io/finalizer
-      - pac.component.appstudio.openshift.io/finalizer
-      name: konflux-multibranch-sample-{{.versionName}}
+        build.appstudio.openshift.io/pipeline: '{"name":"docker-build","bundle":"latest"}'
+        build.appstudio.openshift.io/status: '{"pac":{"state":"enabled","merge-url":"https://github.com/konflux-ci/multi-version-konflux-sample/pull/1","configuration-time":"Wed, 07 Aug 2024 08:59:18 UTC"},"message":"done"}'      
+      name: multi-version-konflux-sample-{{.versionName}}
     spec:
-      application: "konflux-multibranch-sample-{{.versionName}}"
-      componentName: "konflux-multibranch-sample-{{.versionName}}"
+      application: "multi-version-konflux-sample-{{.versionName}}"
+      componentName: "multi-version-konflux-sample-{{.versionName}}"
       source:
         git:
           context: ./
           dockerfileUrl: Dockerfile
           revision: "{{.version}}"
-          url: https://github.com/ifireball/konflux-multibranch-sample.git
+          url: https://github.com/konflux-ci/multi-version-konflux-sample.git
+
+  - apiVersion: appstudio.redhat.com/v1alpha1
+    kind: ImageRepository
+    metadata:
+      name: imagerepository-for-multi-version-konflux-sample-{{.versionName}}
+      labels:
+        appstudio.redhat.com/application: multi-version-konflux-sample-{{.versionName}}
+        appstudio.redhat.com/component: multi-version-konflux-sample-{{.versionName}}
+    spec:
+      image:
+        name: konflux-samples-tenant/multi-version-konflux-sample-{{.versionName}}/multi-version-konflux-sample-{{.versionName}}
+        visibility: public
+      notifications:
+        - config:
+            url: 'https://bombino.preprod.api.redhat.com/v1/sbom/quay/push'
+          event: repo_push
+          method: webhook
+          title: SBOM-event-to-Bombino
+
 ----
 
 The *resources* section for the template may be created by looking at the YAML for existing resources and copying it while removing generated and unnecessary data and adding variable references where needed.
 
+The following kinds of resources are currentyl supported. Please do not attempt to create other kinds as that would prevent the template from being applied successfully.
+
+[cols="2,1,3"]
+|===
+|apiVersion |kind |Comments
+
+|appstudio.redhat.com/v1alpha1
+|Application
+|
+
+|appstudio.redhat.com/v1alpha1
+|Component
+|
+
+|appstudio.redhat.com/v1alpha1
+|ImageRepository
+|Every component should have an accompanying `ImageRepository` resource
+
+|appstudio.redhat.com/v1beta2
+|IntegrationTestScenario
+|Note the `v1beta2` API version. Make sure to use the right version when querying the cluster, otherwise important data may be missing from the resutls you get.
```

Comment:
```suggestion
|Note the `v1beta2` API version. Make sure to use the right version when querying the cluster, otherwise important data may be missing from the results you get.
```